Readme.md

```diff
@@ -138,6 +140,16 @@ a) Directory structure of our current Templates
 COMMANDS FOR RUNNING TESTS
 ---------------------------
 
+Templates for.env files are accessible.
```

Comment:
Good to move this session under Prerequisites or create new session after prerequisites. You may title it as "Configure environmental variables"